client/app/views/account.coffee

```diff
@@ -230,6 +252,15 @@ module.exports = class exports.AccountView extends BaseView
         , ''
     
     
+    # Generate tokens for recovery in case of device loss
+    getRecoveryTokens: () ->
+        tokens = []
+        for [0..9]
+            tempToken = Math.floor Math.random()*100000000
+            tokens.push tempToken
+        tokens
+
```

Comment:
You can safely shorten it with an expression like `tokens = (Math.floor Math.random()*100000000 for [0..9])` :wink: